### PR TITLE
Fixes #12230: Agent check crontab logs stdout

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/12230

### CODE REVIEW

File: techniques/system/common/1.0/rudder-agent-community-cron [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # To temporarily avoid this behaviour, run 'rudder agent disable'
 # Don't forget to remove that file when you're done!
```
Comment (amousset) [inline techniques/system/common/1.0/rudder-agent-community-cron]: this line means nothing, it was before the disable command I guess

File: techniques/system/common/1.0/cron-setup.st [hunk 2]
```diff
@@ -65,11 +65,16 @@ bundle agent setup_cronjob
         edit_defaults => noempty_backup,
         edit_line => delete_lines_matching("0,5,10,15,20,25,30,35,40,45,50,55 \* \* \* \* /usr/bin/test -x /opt/rudder/bin/rudder \&\& /opt/rudder/bin/rudder agent check >/dev/null");
 
+      # Cleanup crontab again
+      "/var/spool/cron/crontabs/root"
+        edit_defaults => noempty_backup,
+        edit_line => delete_lines_matching("0,5,10,15,20,25,30,35,40,45,50,55 * * * * /usr/bin/test -x /opt/rudder/bin/rudder \&\& /opt/rudder/bin/rudder agent check >> /var/log/rudder/agent-ch  eck/check.log 2>\&1");
```
Comment (amousset) [inline techniques/system/common/1.0/cron-setup.st]: should be escaped

Comment (amousset) [inline techniques/system/common/1.0/cron-setup.st]: s/ch  eck/check

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**